packages/itwinui-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -1023,3 +1023,29 @@ it('should update live region when selection changes', async () => {
   await userEvent.click(options[0]);
   expect(liveRegion).toHaveTextContent('Item 1, Item 2');
 });
+
+it('should pass props to statusMessage and endIcon', async () => {
+  const { container, getByText } = render(
+    <ComboBox
+      options={[0, 1, 2].map((value) => ({ value, label: `Item ${value}` }))}
+      message='test message'
+      status='negative'
+      statusMessageProps={{
+        contentProps: { className: 'my-message-content' },
+        iconProps: { className: 'my-message-icon' },
+      }}
+      endIconProps={{ className: 'my-end-icon' }}
```

Comment:
maybe also test for other parts? main className, dropdownMenuProps, and anything else

Reply:
Updated tests